testutil/keeper/params.go

```diff
@@ -0,0 +1,42 @@
+package keeper
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// AssertDefaultParamsEqualExceptFields asserts that the expectedParams and
+// actualParams are equal except for the fields specified in exceptFields.
+// expectedParams and actualParams MUST be reference types (e.g. pionters).
+func AssertDefaultParamsEqualExceptFields[P any](
+	t *testing.T,
+	expectedParams P,
+	actualParams P,
+	exceptFields ...string,
+) {
+	expectedParamsValue := reflect.ValueOf(expectedParams).Elem()
+	actualParamsValue := reflect.ValueOf(actualParams).Elem()
+
+	for fieldIdx := 0; fieldIdx < expectedParamsValue.NumField(); fieldIdx++ {
+		fieldName := expectedParamsValue.Type().Field(fieldIdx).Name
+		if isFieldException(fieldName, exceptFields) {
+			continue
```

Comment:
Shouldn't we require that the fields have different values here?

Reply:
I've added a comment:
![image](https://github.com/pokt-network/poktroll/assets/600733/8429ca6d-d79c-4b52-9406-81287f4d8062)

The idea is that we **don't assert** on fields which are "exceptions to the equality assertion".